examples/nvidia_dali/README.md

```diff
@@ -2,7 +2,7 @@
 
 The NVIDIA Data Loading Library (DALI) is a library for data loading and pre-processing to accelerate deep learning applications. It provides a collection of highly optimized building blocks for loading and processing image, video and audio data.
 
-In this example, we use NVIDIA DALI for pre-processing image input for inference in resnet-18 model.
+In this example, we use NVIDIA DALI for pre-processing image input for inference in mnist model.
```

Comment:
The default DALI handler covers the resnet example so i thought of having mnist model in examples folder so that users can get a better understanding.